From 13b94b3c3271b6b456e5b12d92547077e59e311c Mon Sep 17 00:00:00 2001
From: Bayside308 <nhall0312@gmail.com>
Date: Mon, 15 May 2017 16:17:31 -0400
Subject: [PATCH 66/71] Update pom.xml

I adjusted the file names that are exported by the pom.xml.
---
 rainbow/pom.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/rainbow/pom.xml b/rainbow/pom.xml
index 0a2ac67..82b8c6a 100644
--- a/rainbow/pom.xml
+++ b/rainbow/pom.xml
@@ -20,7 +20,7 @@
                 </property>
             </activation>
             <build>
-                <finalName>Rainbow-${minecraft.version}-b${env.BUILD_NUMBER}</finalName>
+                <finalName>Rails-${minecraft.version}-b${env.BUILD_NUMBER}</finalName>
             </build>
         </profile>
     </profiles>
@@ -130,7 +130,7 @@
     </dependencies>
 
     <build>
-        <finalName>Rainbow-${minecraft.version}</finalName>
+        <finalName>Rails-${minecraft.version}</finalName>
         <resources>
             <resource>
                 <directory>${basedir}/src/main/resources</directory>
@@ -281,7 +281,7 @@
                             <goal>single</goal>
                         </goals>
                         <configuration>
-                            <finalName>Rainbow-${minecraft.version}-b${build.number}</finalName>
+                            <finalName>Rails-${minecraft.version}-b${build.number}</finalName>
                             <attach>false</attach>
                             <appendAssemblyId>false</appendAssemblyId>
                             <descriptors>
-- 
2.10.1.windows.1

